tests/cypress/support/helpers/minting-mocks.ts

```diff
@@ -4,6 +4,6 @@ export const mockMintMarkets = () =>
   })
 
 export const mockMintSnapshots = () =>
-  cy.intercept('https://prices.curve.finance/v1/crvusd/markets/*/*/snapshots?agg=none&fetch_on_chain=true', {
+  cy.intercept('https://prices.curve.finance/v1/crvusd/markets/*/*/snapshots*', {
```

Comment:
We should get errors if the URL changes by mistake
```suggestion
  cy.intercept('https://prices.curve.finance/v1/crvusd/markets/*/*/snapshots?fetch_on_chain=true', {
```